datasetinsights/constants.py

```diff
@@ -53,3 +53,6 @@
     "allow_extra_args": True,
 }
 DEFAULT_DATASET_VERSION = "latest"
+
+MLFLOW_TRACKER = "mlflow"
+DEFAULT_RUN_NAME = "run_" + str(datetime.now())
```

Comment:
Can this name be more specific as to which run it is referring to?

Reply:
Fixed